addons/mail/static/src/components/chat_window_hidden_menu/chat_window_hidden_menu.js

```diff
@@ -53,7 +53,9 @@ export class ChatWindowHiddenMenu extends Component {
             return;
         }
         this._applyListHeight();
-        this._applyOffset();
+        if (this.root.el) {
+            this._applyOffset();
+        }
```

Comment:
arguable that it could even be moved at the top instead (if there's no root, _listRef in _applyListHeight will also crash for now)

Reply:
or even on the parent of this too (calling _apply making no sense)